<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PQueue</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources/images.qrc">
    <normaloff>:/images/icon64.png</normaloff>:/images/icon64.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>459</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>PJobRunner</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QListWidget" name="pjobRunnerListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Network interface:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="networkInterfaceComboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="startScanButton">
          <property name="text">
           <string>Start scan</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="poolStatusLabel">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Jobs</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Playlist</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>6</number>
             </property>
             <item>
              <widget class="QListWidget" name="jobsWidget">
               <property name="selectionMode">
                <enum>QAbstractItemView::ExtendedSelection</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <spacer name="verticalSpacer_5">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>18</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="jobUpButton">
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>up</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="jobDownButton">
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>down</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_3">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="jobDeleteButton">
                 <property name="maximumSize">
                  <size>
                   <width>40</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>-</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="stopButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
               <property name="icon">
                <iconset resource="Resources/images.qrc">
                 <normaloff>:/images/stop.png</normaloff>:/images/stop.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="startButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
               <property name="icon">
                <iconset resource="Resources/images.qrc">
                 <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Parameters:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="parametersWidget"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addJobButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Add to playlist</string>
              </property>
              <property name="default">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Results</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QComboBox" name="pjobFileSelector"/>
      </item>
      <item row="1" column="0" colspan="4">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableView" name="resultView"/>
        </item>
       </layout>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="loadPreviousRunsButton">
        <property name="text">
         <string>Load runs from previous simulations</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="Seite">
     <attribute name="title">
      <string>Visualization</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QSplitter" name="visualizationHorizontalSplitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="visualizationBox">
         <property name="title">
          <string>Visualization</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QCheckBox" name="showInterpolant">
            <property name="text">
             <string>show interpolation function</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QSplitter" name="visualizationVerticalSplitter">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Results</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <property name="margin">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <widget class="QTreeWidget" name="navigatorResultsTreeWidget">
             <column>
              <property name="text">
               <string>1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="parametersBox">
          <property name="title">
           <string>Parameters</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="logDockWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>590</y>
      <width>181</width>
      <height>232</height>
     </rect>
    </property>
    <property name="floating">
     <bool>false</bool>
    </property>
    <property name="windowTitle">
     <string>Log</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QTextEdit" name="logTextEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="scriptingConsoleDockWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>470</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
    <property name="windowTitle">
     <string>Scripting console</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_3">
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QTextEdit" name="scriptOutput">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="scriptLineEdit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QDockWidget" name="scriptFilesDockWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>346</width>
      <height>191</height>
     </rect>
    </property>
    <property name="windowTitle">
     <string>Scripts</string>
    </property>
    <widget class="QWidget" name="dockWidgetContents_4">
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <widget class="QListWidget" name="scriptsListWidget"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QProgressBar" name="scriptProgressBar">
          <property name="maximum">
           <number>0</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
          <property name="textVisible">
           <bool>false</bool>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runScriptButton">
          <property name="text">
           <string>Run script</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopScriptButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Stop script</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionMruAction1"/>
    <addaction name="actionMruAction2"/>
    <addaction name="actionMruAction3"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuResults">
    <property name="title">
     <string>Results</string>
    </property>
    <addaction name="actionImport_From_PJob"/>
    <addaction name="actionImport_From_CSV"/>
    <addaction name="actionExport_To_CSV"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuScripting">
    <property name="title">
     <string>Scripting</string>
    </property>
    <addaction name="actionLoad_script_from_file"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
    <addaction name="actionShow_log"/>
    <addaction name="actionShow_scripting_console"/>
    <addaction name="actionShow_loaded_scripts"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuResults"/>
   <addaction name="menuScripting"/>
   <addaction name="menuSettings"/>
   <addaction name="menuWindows"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_create_job_window">
   <property name="text">
    <string>Show create job window</string>
   </property>
  </action>
  <action name="actionNew_Parametervariation">
   <property name="text">
    <string>New Parametervariation...</string>
   </property>
  </action>
  <action name="actionRun_Queue">
   <property name="text">
    <string>Run Queue</string>
   </property>
  </action>
  <action name="actionExport_To_CSV">
   <property name="text">
    <string>Export To CSV...</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShow_log">
   <property name="text">
    <string>Show log</string>
   </property>
  </action>
  <action name="actionShow_scripting_console">
   <property name="text">
    <string>Show scripting console</string>
   </property>
  </action>
  <action name="actionLoad_script_from_file">
   <property name="text">
    <string>Load script from file...</string>
   </property>
  </action>
  <action name="actionShow_loaded_scripts">
   <property name="text">
    <string>Show loaded scripts</string>
   </property>
  </action>
  <action name="actionMruAction1">
   <property name="text">
    <string>mruAction1</string>
   </property>
  </action>
  <action name="actionMruAction2">
   <property name="text">
    <string>mruAction2</string>
   </property>
  </action>
  <action name="actionMruAction3">
   <property name="text">
    <string>mruAction3</string>
   </property>
  </action>
  <action name="actionImport_From_PJob">
   <property name="text">
    <string>Import From PJob...</string>
   </property>
  </action>
  <action name="actionImport_From_CSV">
   <property name="text">
    <string>Import From CSV...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>403</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_log</sender>
   <signal>triggered()</signal>
   <receiver>logDockWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>764</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_scripting_console</sender>
   <signal>triggered()</signal>
   <receiver>scriptingConsoleDockWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>550</x>
     <y>611</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow_loaded_scripts</sender>
   <signal>triggered()</signal>
   <receiver>scriptFilesDockWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>746</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
